ci/test_integration.sh

```diff
@@ -1,9 +1,20 @@
 #!/usr/bin/env bash
 
+set -euo pipefail
+
+if [ -z ${LD_LIBRARY_PATH+x} ]; then
+    LD_LIBRARY_PATH=$(pwd)/chainer_ctc/ext/warp-ctc/build
+else
+    LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:$(pwd)/chainer_ctc/ext/warp-ctc/build
+fi
+
+export LD_LIBRARY_PATH
```

Comment:
It is better to add some comments here. (`path.sh`)
And you can use `shellcheck disable=SC...` to simplify

```bash
# NOTE: LD_LIBRARY_PATH is updated in `path.sh` but does not work. (Maybe?)
# shellcheck disable=SC<shell_check_code_number>
export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:$(pwd)/chainer_ctc/ext/warp-ctc/build

```

Reply:
The problem about D_LIBRARY_PATH is that it adds the chainer_ctc in tools (by default in pc env) but in ci it is install only in `pwd` (no tools) that is why raises error in Ubuntu tests.

I will disable SC for LD_LIBRARY_PATH

Reply:
I see.
Another choice is as follows:
https://github.com/espnet/espnet/blob/6759e7fcb8386b845a8a48ef779db3977f599a05/ci/test_python.sh#L13

Reply:
I think fixing `install.sh` to put `chainer_ctc` in `tools/` is better because it is closer to our usage.